### PR TITLE
Enarx info enarx version

### PR DESCRIPTION
**Pull request by moksh-pathak**
Closes Issue #1208

Enarx version is printed whenever `enarx info` or `enarx info --json` is run on terminal.
Made changes in src/cli/info.rs
Made a struct Info, which is used when the `--json` flag is true. It prints the version no. and the backend.
When `--json` is not used, version no. is printed using `env!("CARGO_PKG_VERSION")`.

### CODE REVIEW

File: src/cli/info.rs [hunk 1]
```diff
@@ -20,8 +27,14 @@ impl Options {
...
+            let info = Info {
+                version: env!("CARGO_PKG_VERSION"),
+                backends,
+            };
+
+            println!("{}", serde_json::to_string_pretty(&info)?);
         } else {
+            println!("Version {} starting up", env!("CARGO_PKG_VERSION"));
             for backend in backends {
                 println!("Backend: {}", backend.name());
```
Comment (haraldh) [inline src/cli/info.rs:39]: You could implement `Display` also for `Info` and use it in both paths.

**Review by haraldh — CHANGES_REQUESTED** [review]

**Comment by npmccallum** [comment]
@moksh-pathak Could you paste an example of the json output?

File: src/cli/info.rs (hunk 1 above)
Comment (moksh-pathak) [reply src/cli/info.rs:39]: I shall look into that.

**Comment by moksh-pathak** [comment]
@npmccallum the example of json output is as follows
Screenshot from 2022-02-03 10-13-39

and the example of non json output is as follows
Screenshot from 2022-02-03 10-14-19


Hope this helps!

**Comment by haraldh** [comment]
Lint is failing... https://github.com/enarx/enarx/runs/5068774406?check_suite_focus=true

**Comment by haraldh** [comment]
please run `cargo fmt` before pushing

**Comment by moksh-pathak** [comment]
> please run `cargo fmt` before pushing

Thanks for the suggestion @haraldh , same has been noted and implemented.

**Comment by haraldh** [comment]
If the tests pass, you want to squash your commits.

**Review by haraldh — CHANGES_REQUESTED** [review]
Please squash your commits to one, as it is small enough and corrections to previous commits should be squashed into them.

**Comment by moksh-pathak** [comment]
Thanks for merging my PR @haraldh :)

**Pull request merged**